libs/host/GarnetServer.cs

```diff
@@ -59,6 +59,10 @@ static string GetVersion()
         /// </summary>
         protected StoreWrapper storeWrapper;
 
+        // IMPORTANT: Keep the version in sync with .azure\pipelines\azure-pipelines-external-release.yml line ~6,
+        // charts\garnet\Chart.yaml line ~6 and increment line ~5 and generate charts\garnet\README.md with helm-docs.
+        readonly string version = "1.0.32";
```

Comment:
merge with latest from main to use the new format of version

Reply:
Fixed by 3b838b4fd9fcfe785ceeb9a6ae52a36c4ddbfcb5